crates/common/src/da.rs

```diff
@@ -37,3 +42,64 @@ pub async fn get_da_block_at_height<Da: DaService>(
         .put(l1_block.header().height(), l1_block.clone());
     Ok(l1_block)
 }
+
+pub fn extract_sorted_sequencer_commitments<Da>(
```

Comment:
I am not sure we need a separate `extract_sorted_sequencer_commitments` if we can have `extract_sequencer_commitments` and then sort if afterwards.